subsys/net/lib/shell/sockets.c

```diff
@@ -123,10 +123,7 @@ static void walk_socket_services(const struct net_socket_service_desc *svc,
 	snprintk(owner, sizeof(owner), "<unknown>");
```

Comment:
The commit subject can be optimized a bit, no need to mention so much stuff so
```
net: shell: sockets: Don't show service type
```
is enough.